scripts/deploy_docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+PROJECT_ID=lightstep
+DATETIME=$(date "+%Y-%m-%d_%H-%M-%S")
+BASENAME=lightstep-benchmark
+PROJECT_ID=$(gcloud config --format text list project | cut -d' ' -f2)
+
+NAME=gcr.io/${PROJECT_ID}/${BASENAME}
+
+docker build --no-cache -t ${NAME}:${DATETIME} ${GOPATH}/../docker
+docker tag -f ${NAME}:${DATETIME} ${NAME}:latest
+
+gcloud docker push ${NAME}:${DATETIME}
```

Comment:
Not sure what your plans for the scope of this is, but you don't need to `push` if you are going to `run` locally.